.rubocop.yml

```diff
@@ -29,6 +29,10 @@ Bundler/OrderedGems:
 RSpec/DescribeClass:
   Enabled: false
 
+# https://github.com/bbatsov/rubocop/issues/3629
```

Comment:
Could you also add a "NOTE: [Rails 5]" stating that this is no longer needed there ?